index.js

```diff
@@ -89,6 +89,7 @@ const createClient = (profile, userAgent, request = _request) => {
 		.then((d) => {
 			if (!Array.isArray(d.jnyL)) return []
 			const parse = parser(profile, opt, {
+				raw: d,
```

Comment:
It seems a bit inconsistent to pass all of `d` here instead of `d.common`

Reply:
True. My intention is to migrate all `profile.parse*` functions to just take `d` though; I didn't want to change all in this PR.

Reply:
k